docs/config/dfget_properties.md

```diff
@@ -0,0 +1,41 @@
+# Customize dfget properties
+
+This topic explains how to customize the dragonfly dfget startup parameters.
+
+**NOTE**: By default, dragonfly config files locate `/etc/dragonfly`. You can create `dfget.yml` for configing dfget startup params. When deploying with Docker, you can mount the default path when starting up dfdaemon image with `-v`.
+
+```sh
+docker run -d  --net=host --name supernode --restart=always -v /etc/dragonfly:/etc/dragonfly dragonflyoss/supernode:0.4.3 --advertise-ip=10.131.160.103
```

Comment:
Is this example right? I think we should use dfclient example here.

Reply:
yep, i just want to give a example for explaining to mount configuration director with `-v`. Maybe modify ip=127.0.0.1 for better?

Reply:
I think maybe you understood the meaning of @yeya24. In fact, we don't recommend the users to run dfget in the docker. So maybe it's not necessary for dfget to write an example to mount configuration director with docker. Could you please just delete related content? THX.